src/main/resources/templates/placeOrder.html

```diff
@@ -17,121 +17,121 @@
             <h1 class="display-4">Place an Order</h1>
         </div>
     </div>
-</main>
 
-<div class="container">
-    <form>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inlineFormInputName">Name</label>
-                <input type="email" class="form-control" id="inlineFormInputName" placeholder="Company or name">
+    <div class="container outerContentWrapper">
+        <form action="#" th:action="@{/saveOrder}" th:object="${saveOrder}" method="post">
```

Comment:
Added the attributes to the form elements, action, th:action, th:object, method.